guide/12-enrich-data-with-thematic-information/part7_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>Our analysis starts with identifying the country we are going to be working with and instantiating an <code>arcgis.geoenrichment.Country</code> object referencing an <code>arcgis.gis.GIS</code> source to use for analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>